test/integration/error.test.js

```diff
@@ -38,13 +41,14 @@ describe(Support.getTestDialectTeaser('Sequelize Errors'), function () {
       expect(validationError).to.be.instanceOf(Sequelize.ValidationError);
       expect(validationError).to.be.instanceOf(Error);
       expect(validationError).to.have.property('name', 'SequelizeValidationError');
-      expect(validationError.message).to.match(/notNull Violation: <field name> cannot be null,\nstring violation: <field name> cannot be an array or an object/);
+      expect(validationError.message).to.equal(errorMessage);
```

Comment:
This change is needed since the error message is passed in upon creation in this particular case, hence will be used instead of the concatenated message. I added an additional test to test for the concatenated message in case no explicit error message is given.